src/client/play.js

```diff
@@ -19,7 +19,7 @@ module.exports = function (client, options) {
         uuid: uuid.v4fast()
       }
 
-      client.write('session', {
+      client.write('chat_session_update', {
```

Comment:
why did we rename this packet ?

Reply:
The auth/microsoftAuth code emit the "session" event after authenticating, so it's not possible to disambiguate in event handler if it's the auth session or packet session. So I updated the naming in mcdata to match the vanilla packet name